packages/nodes-base/nodes/SyncroMSP/v1/actions/contact/add/description.ts

```diff
@@ -0,0 +1,81 @@
+import {
+	ContactProperties,
+} from '../../Interfaces';
+
+import {
+	addressFixedCollection
+} from '../../../methods/sharedFields';
+
+export const contactAddDescription: ContactProperties = [
+	{
+		displayName: 'Customer ID',
+		name: 'customerId',
```

Comment:
For other ones like Contacts does just "id" get used. We should probably do the same here.